cardano-config/cardano-config.cabal

```diff
@@ -42,14 +45,17 @@ library
                      , optparse-applicative
                      , ouroboros-consensus
                      , ouroboros-network
+                     , process
```

Comment:
why is this new dependency needed?

Reply:
Because git revision fetching was moved from `cardano-node` to `cardano-config`.